modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -0,0 +1,249 @@
+import Panel from 'Panel';
+/**
+ * New Profile Form
+ *
+ * Create a new profile form
+ *
+ * @author  Shen Wang
+ * @version 1.0.0
+ * */
+class NewProfileIndex extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: {},
+            configData: {},
+            formData: {},
+            newData: {},
+            isLoaded: false,
+            isCreated: false,
+            errMessage: '',
+        };
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.setFormData = this.setFormData.bind(this);
+        this.fetchData = this.fetchData.bind(this);
+    }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   * Additionally add hiddenHeaders to global loris variable
+   * for easy access by columnFormatter.
```

Comment:
I thought hiddenHeaders was removed? Also, the fetchData function doesn't seem to do that anyways..

Reply:
done